bridge-token/src/lib.rs

```diff
@@ -98,8 +111,7 @@ impl BridgeToken {
 
     #[payable]
     pub fn withdraw(&mut self, amount: U128, recipient: String) -> Promise {
-        self.check_not_paused(PAUSE_WITHDRAW);
-
+        require!(!self.is_paused());
```

Comment:
Why we don't use the near plugins for this contract?

Reply:
The token contract can be controlled just by the `Factory` contract, it was possible to do that when plugins like the Pausable plugin were implemented based on Ownable, but now the near plugins use ACL everywhere which complicates the maintenance and the implementation.